client/webserver/site/src/css/icons.scss

```diff
@@ -110,6 +110,11 @@
   transform: rotate(180deg);
 }
 
+.ico-wide-headed-left-arrow::before {
+  content: "\e919";
```

Comment:
You forgot to add the `<glyph>` to the SVG file

Reply:
Dang. I've updated the SVG file and re-generated the font files.